tests/restapi/conftest.py

```diff
@@ -55,3 +55,92 @@ def restrict_sqlalchemy_queuepool(aiida_profile):
     backend_manager = get_manager().get_backend_manager()
     backend_manager.reset_backend_environment()
     backend_manager.load_backend_environment(aiida_profile, pool_timeout=1, max_overflow=0)
+
+
+@pytest.fixture
+def populate_restapi_database(clear_database_before_test, aiida_localhost):
+    """Populates the database with a considerable set of nodes to test the restAPI"""
+    # pylint: disable=too-many-locals, too-many-statements, unused-argument
+    import tempfile
+    from aiida import orm
+    from aiida.common.links import LinkType
+
+    # create test inputs
+    cell = ((2., 0., 0.), (0., 2., 0.), (0., 0., 2.))
+    structure = orm.StructureData(cell=cell)
+    structure.append_atom(position=(0., 0., 0.), symbols=['Ba'])
+    structure.store()
+    structure.add_comment('This is test comment.')
+    structure.add_comment('Add another comment.')
```

Comment:
why do you need all these comments and attributes/extras you set on the calculations further down, as well as the `Log` entries? They are not involved in the actual test on the namespace tree whatsoever